### PR TITLE
Add suggestion to use `set -e`.

### PR DESCRIPTION
**Pull request by bbewan**
https://app.asana.com/0/353548986073982/467608476540281

### CODE REVIEW

File: builds/custom_build_steps.adoc [hunk 1]
```diff
@@ -414,7 +400,28 @@ if [[ "$BUDDYBUILD_BRANCH" =~ "release" ]]; then
...
+----
+#!/usr/bin/env bash
+
+set -e
+
+grep "ERROR" *.logs
+scp *.logs user@remote:failures/
+----
+
+This example searches for `ERROR` in log files. If there are no error
```
Comment (clintam) [inline builds/custom_build_steps.adoc]: sort of weird example: the build errors if there are no ERRORs in the logs.

I wonder if we should just add set -e to an existing examples.

Comment (bbewan) [reply builds/custom_build_steps.adoc]: I've revised the example, to avoid the weirdness.

**Comment by bbdocs** [comment]
See the website here: http://106-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://106-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://106-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://106-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://106-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://106-bb-docs.surge.sh

**Pull request merged**